src/gl_context.rs

```diff
@@ -47,8 +48,10 @@ impl<Native> GLContext<Native>
         };
 
         try!(native_context.make_current());
+        let extensions = gl_.get_string(gl::EXTENSIONS);
+        let extensions: Vec<String> = extensions.split(&[',',' '][..]).map(|s| s.into()).collect();
```

Comment:
I didn't use Vec<&str> in this case because the String returned from gl.get_string(gl::EXTENSIONS) is destroyed. We'd need to save it somewhere to have a Vec<&str>